### PR TITLE
8298038: [s390] Configure script detects num_cores +1

### PR DESCRIPTION
**Pull request by offamitkumar**
cpuinfo on s390x architecture contains additional header information on Linux/Z which is not seen on GNU Linux:

```
vendor_id       : IBM/S390
# processors    : 8
```
due to this one extra build job is created. 

Reported Issue can be found here : [JDK-8298038](https://bugs.openjdk.org/browse/JDK-8298038)

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8298038](https://bugs.openjdk.org/browse/JDK-8298038): [s390] Configure script detects num_cores +1

### Reviewers
 * [Tyler Steele](https://openjdk.org/census#tsteele) (@backwaterred - Committer)
 * [Erik Joelsson](https://openjdk.org/census#erikj) (@erikj79 - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11659/head:pull/11659` \
`$ git checkout pull/11659`

Update a local copy of the PR: \
`$ git checkout pull/11659` \
`$ git pull https://git.openjdk.org/jdk pull/11659/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11659`

View PR using the GUI difftool: \
`$ git pr show -t 11659`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11659.diff">https://git.openjdk.org/jdk/pull/11659.diff</a>

</details>

### CODE REVIEW

**Review by backwaterred — APPROVED** [review]

**Comment by backwaterred** [comment]
LGTM. You'll need a review from a **R**eviewer as well.

Congrats on opening your first PR :tada:.

**Comment by offamitkumar** [comment]
Waiting for OCA approval.

**Review by erikj79 — APPROVED** [review]

**Review by backwaterred — APPROVED** [review]

**Pull request closed without merging**
